deployment/cfn/follower.py

```diff
@@ -0,0 +1,187 @@
+"""Mesos Follower Node"""
+
+from troposphere import (
+    Parameter,
+    Tags,
+    Ref,
+    Base64,
+    ec2)
+
+import template_utils as utils
+import troposphere.autoscaling as asg
+
+from majorkirby import StackNode
```

Comment:
Going to open up a bunch of issues on the `majorkirby` repository so that we can start the progress of getting this out to PyPI.